pynestml/codegeneration/nest_code_generator.py

```diff
@@ -619,6 +628,12 @@ def _get_neuron_model_namespace(self, neuron: ASTNeuron) -> Dict:
                     marks_delay_vars_visitor = ASTMarkDelayVarsVisitor()
                     expr_ast.accept(marks_delay_vars_visitor)
 
+                # Check if the update expressions have vector variables and update the vector parameters
```

Comment:
can this be done somewhere more meaningful than the ``_get_neuron_model_namespace()`` method?